dotcom-rendering/src/lib/getFrontsAdPositions.ts

```diff
@@ -9,10 +13,66 @@ type GroupedCounts = {
 	standard: number;
 };
 
+type AdCandidate = Pick<DCRCollectionType, 'collectionType'>;
+
+const getMerchHighPosition = (collectionCount: number): number =>
+	collectionCount >= 4 ? 2 : 0;
+
+// This happens on the recipes front, where the first container is a thrasher see: https://github.com/guardian/frontend/pull/20617
```

Comment:
Tiny nitpick but can we convert this to a TSDoc comment?

```suggestion
/**
  * This happens on the recipes front, where the first container is a thrasher
  * @see https://github.com/guardian/frontend/pull/20617
  */
```

Reply:
Exactly 5 years on from that PR and the comment still holds true!